Makefile

```diff
@@ -2,24 +2,24 @@ all: build release clean
 
 VERSION=`git tag |tail -n1`
 build:
-	@cd src && go build -ldflags "-s -w -X main.version=${VERSION}" .
+	@go build -ldflags "-s -w -X $(shell go list)/internal.Version=${VERSION}" ./cmd/gorss/...
```

Comment:
cmd doesn't exists here. Not sure what the intent is? Why build to an `cmd/gorss` directory?

Reply:
Fixed :) sorry! got missed by `.gitignore`